### PR TITLE
en, zh: add tikv-ctl in tls document

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add tikv-ctl in TLS document.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/314
- Other reference link(s):

### CODE REVIEW

File: zh/enable-tls-between-components.md [hunk 1]
```diff
@@ -1058,9 +1058,9 @@ category: how-to
 
     参考官网文档：[Download TiDB installation package](https://pingcap.com/docs/v3.0/reference/tools/pd-control/#download-tidb-installation-package)。
 
-2. 连接集群。
+2. 下载 Client 端证书。
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md:1061]: @weekface PTAL if my modification is correct. Thanks!

Comment (weekface) [reply zh/enable-tls-between-components.md:1061]: good catch

**Pull request merged**